pandas/io/formats/style.py

```diff
@@ -1368,8 +1419,19 @@ def set_table_attributes(self, attributes: str) -> Styler:
         Examples
         --------
         >>> df = pd.DataFrame(np.random.randn(10, 4))
-        >>> df.style.set_table_attributes('class="pure-table"')
-        # ... <table class="pure-table"> ...
+        >>> df.style.set_table_attributes('class="pure-table"')  # doctest: +SKIP
+               0      1      2      3
+        0    0.4    1.0   -1.1    0.5
+        1   -0.6    1.2    1.8   -1.4
+        2    1.2   -0.6   -0.2   -0.2
+        3   -0.5   -0.8   -0.2    1.8
+        4   -0.5   -0.8    0.2   -1.7
+        5    0.1    1.5    0.3   -1.3
+        6    1.5   -0.6   -1.3   -0.1
+        7   -0.1   -0.8   -0.7    0.3
+        8    1.7   -0.5   -1.4    0.8
+        9    1.5    0.4   -0.8    0.7
+
```

Comment:
i think this is a distractor as well. You have deleted the output which shows the addition to the HTML string, which is the point of the method, and replaced it with a table output which doesn't really add any value to the documentation. To be fair, maybe the definition of the `df` is also a distractor and the example could be cleaner by assuming that `df` already exists - also makes it more generic.

Reply:
So, I'll leave it as it was before. If anyone or you have any better suggestions, then just adjust from what was previously.